### ISSUE TITLE
Failed To Load Tensorflow Runtime And Import Error dll Load Failed

### ISSUE THREAD

**Issue by Bhanu1434** [iperov/DeepFaceLab#5402]
I Have Never used Deepfacelab before tried to make my first test deepfake with the help of online tutorials. But when i tried to run
2) extract images from video data_src.bat
got a 4 pargraph long error (attached below)

I'm using DFL latest DirectX12 

![Screenshot 2021-10-04 194409](https://user-images.githubusercontent.com/91895672/135867392-5c0477af-a3cd-4490-9395-eca28244c512.png)
![Screenshot 2021-10-04 194444](https://user-images.githubusercontent.com/91895672/135867399-69ac052d-e526-4c84-b225-94e9ce1e5264.png)
version

**Comment by SKEMAM** [comment]
Have you find this solution?

**Comment by Bhanu1434** [comment]
> Have you find this solution?

Nope... Not Exactly But I did find out that this error is generated because of my processor. DFL needs "AVX or AVX2" instructions support which my processor didn't had. So I used  DFL colab for everything From extraction to merging of frames And For Xseg I used pretrained Xseg models. But Now I've changed my processor And DFL works Fine. Still I use colab Because I don't have GPU.
So I guess solution is : you have to upgrade your processor.

> Have you find this solution?

> Have you find this solution?

Nope... Not Exactly But I did find out that this error is generated because of my processor. DFL needs "AVX or AVX2" instructions support which my processor didn't had. So I used  DFL colab for everything From extraction to merging of frames And For Xseg I used pretrained Xseg models. But Now I've changed my processor And DFL works Fine. Still I use colab Because I don't have GPU.
So I guess solution is : you have to upgrade your processor.

**Comment by SKEMAM** [comment]
Oh. Thank you.
